projects/packages/forms/src/contact-form/js/date-picker.js

```diff
@@ -0,0 +1,18 @@
+import { DatePicker } from './../libs/date-picker/date-picker';
+
+document.addEventListener( 'DOMContentLoaded', () => {
+	document.querySelectorAll( '.jp-contact-form-date' ).forEach( function ( node ) {
+		DatePicker( node, {
+			lang: window.jpDatePicker.lang,
+			dayOffset: Number( window.jpDatePicker.offset ),
+			dateFormat: node.dataset.format,
+			hasFooter: false,
+		} ).on( 'close', function () {
+			var event = new Event( 'blur', {
```

Comment:
Does this file get built?

Assuming it does, you could use `const` and array functions `() => {}`